src/main/java/edu/hm/hafner/analysis/parser/AquaScannerParser.java

```diff
@@ -76,12 +73,11 @@ private Severity mapSeverity(final String string) {
         else if (StringUtils.containsAnyIgnoreCase(string, AQUA_VULNERABILITY_LEVEL_TAG_MEDIUM)) {
             return Severity.WARNING_NORMAL;
         }
-        else if (StringUtils.containsAnyIgnoreCase(string, AQUA_VULNERABILITY_LEVEL_TAG_HIGH, AQUA_VULNERABILITY_LEVEL_TAG_CRITICAL,
-                AQUA_VULNERABILITY_LEVEL_TAG_MALWARE, AQUA_VULNERABILITY_LEVEL_TAG_SENSITIVE)) {
+        else if (StringUtils.containsAnyIgnoreCase(string, AQUA_VULNERABILITY_LEVEL_TAG_HIGH)) {
```

Comment:
For a single argument it makes more sense to use `String.isEqualToIgnoreCase`

Reply:
Fixed